src/select/select.ts

```diff
@@ -259,6 +259,7 @@ export class SelectComponent implements OnInit, ControlValueAccessor {
   @Input() public textField:string = 'text';
   @Input() public childrenField:string = 'children';
   @Input() public multiple:boolean = false;
+  @Input() public formControl = new FormControl(); 
```

Comment:
Sorry I was trying to see how you would use a form control with a ControlValueAccessor and I don't see any examples like that? Would you happen to have one? I tried using a form control like usual with [formControl] and [formControlName].